### ISSUE TITLE
Feature Request: --iron-man-using-gauntlet flag

### ISSUE THREAD

**Issue by codekidX** [anandundavia/thanos-js#8]
need to restore back the deleted files at the exact same place and time that it was deleted.

**Comment by codekidX** [comment]
@nipunbharti21 The way you worked it around looks promising, but the side effects of using this flag also need to be handled _(i.e: this flag cannot be used anymore)_

**Comment by nipunbharti21** [comment]
Yes, that's true. Sorry, I forgot about this detail as **SPOILER ALERT** Iron Man dies and the infinity stones are returned back to their respected spots. I'll definitely add this feature and raise a pull request.
Thank you for the reminder :)

**Comment by nipunbharti21** [comment]
@codekidX Added that side effect as well

**Comment by arielmol** [comment]
For recovering people you need to go back in time, so you need git, git is the time stone.
